docker/mainnet/bitcoin-cash/bitcoin-cash.mainnet.conf

```diff
@@ -0,0 +1,15 @@
+port=9333
+
+rpcbind=0.0.0.0
+rpcport=9332
+
+rpcallowip=172.100.2.100
+
+txindex=1
+usecashaddr=0
```

Comment:
Is that somehow affects type of addresses user could use?

Reply:
it shouldn't, it is just about generating new address on local wallet. But we still have problems with new address types validation.